distributed/scheduler.py

```diff
@@ -5415,6 +5415,8 @@ async def handle_worker(self, comm: Comm, worker: str) -> None:
         logger.info("Starting worker compute stream, %s", worker)
         try:
             await self.handle_stream(comm=comm, extra={"worker": worker})
+        except CommClosedError:
+            logger.info("Connection to worker %s has been closed.", comm.peer_address)
```

Comment:
exception level + reraise?

Reply:
Why not reraise here?

Reply:
My bad, apparently I forgot to push that change. It's there now.

Reply:
Superseded by https://github.com/dask/distributed/pull/7162#discussion_r1011638692